stix_shifter_modules/msatp/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -92,43 +412,92 @@
         "object": "nc_process_file"
       },
       {
-        "key": "process.image_ref",
+        "key": "process.binary_ref",
```

Comment:
it should be `process.image_ref`. stix 2.1 doesn't have a binary_ref property for process object.

Reply:
thanks, fixed